### PR TITLE
Classes with Random colores and parameters

### PR DESCRIPTION
**Pull request by misterSokor**
created classes with Random parameters

### CODE REVIEW

File: src/main/java/core/basesyntax/figure/type/Rectangle.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figure.type;
+
+public class Rectangle extends Figures {
+
+    private int side1;
+    private int side2;
```
Comment (aaananas) [inline src/main/java/core/basesyntax/figure/type/Rectangle.java:6]:
```suggestion
    private int firstSide;
    private int secondSide;
```

File: src/main/java/core/basesyntax/suppliers/ColoreSupplier.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.figure.type.Colors;
+import java.util.Random;
+
+public class ColoreSupplier {
+    public String getRandomColor() {
+        int colorIndex = new Random().nextInt(Colors.values().length - 1);
```
Comment (aaananas) [inline src/main/java/core/basesyntax/suppliers/ColoreSupplier.java:8]:
```suggestion
public class ColoreSupplier {
    private static final Random random = new Random();
    
    public String getRandomColor() {
        int colorIndex = random.nextInt(Colors.values().length - 1);
```

File: src/main/java/core/basesyntax/suppliers/FigureSupplier.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import core.basesyntax.figure.type.Rectangle;
+import core.basesyntax.figure.type.RightTriangle;
+import core.basesyntax.figure.type.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColoreSupplier coloreSupplier = new ColoreSupplier();
+
+    public Figures getRandomFigure() {
+        int figureIndex = new Random().nextInt(4);
```
Comment (aaananas) [inline src/main/java/core/basesyntax/suppliers/FigureSupplier.java:15]:
```suggestion
    private ColoreSupplier coloreSupplier = new ColoreSupplier();
    private Random random = new Random();
    
    public Figures getRandomFigure() {
        int figureIndex = random.nextInt(4);
```
lets make random global

File: src/main/java/core/basesyntax/suppliers/FigureSupplier.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        switch (figureIndex) {
+            case 0 :
+                return new Circle(coloreSupplier.getRandomColor(), getRandomInt());
```
Comment (aaananas) [inline src/main/java/core/basesyntax/suppliers/FigureSupplier.java:18]:
```suggestion
                return getRandomCircle();
```
make methods that create each random figure

**Review by aaananas — CHANGES_REQUESTED** [review]

**Review by zachar-chorny — CHANGES_REQUESTED** [review]
Screen Shot 2022-09-21 at 9 13 08 PM
Fix it

**Pull request closed without merging**